src/vs/editor/contrib/goToDeclaration/browser/goToDeclarationCommands.ts

```diff
@@ -279,6 +280,13 @@ export class GoToImplementationAction extends ImplementationAction {
 			}
 		});
 	}
+
+	public run(accessor: ServicesAccessor, editor: editorCommon.ICommonCodeEditor): TPromise<void> {
```

Comment:
I'd say move it to `DefinitionAction`. That way, `go to definition` and `go to type definition` also pick it up